src/test/java/org/kohsuke/github/ArchTests.java

```diff
@@ -19,6 +23,17 @@
             .withImportOption(new ImportOption.DoNotIncludeJars())
             .importPackages("org.kohsuke.github");
 
+    private static final DescribedPredicate<JavaAnnotation<?>> previewAnnotationWithNoMediaType = new DescribedPredicate<JavaAnnotation<?>>(
```

Comment:
This predicate ensures that no usages of the preview annotation will accept an empty array for the value param.